README.md

```diff
@@ -27,6 +28,22 @@ To reset the environment, clean up with `python manage.py clean`.
 6. Run the test suite with `python manage.py test`
 
 ## Deployment
-As an example of deployment, spkrepo provides [SaltStack](http://www.saltstack.com/) files in the `salt` directory.
-You can test this with [Vagrant](https://www.vagrantup.com/) by running `vagrant up`. This makes easy to reproduce
-production-like environments.
+
+https://flask.palletsprojects.com/en/1.1.x/tutorial/deploy/
+
+Modify `spkrepo/config.py` to disable debug logs, set a secure key and optionally set a cache:
```

Comment:
Actually you should define SPKREPO_CONFIG=/path/to/config.py it will use the default config and override only what you put there.